app/com/linkedin/drelephant/spark/fetchers/SparkFetcher.scala

```diff
@@ -41,6 +41,7 @@ class SparkFetcher(fetcherConfigurationData: FetcherConfigurationData)
   import ExecutionContext.Implicits.global
 
   private val logger: Logger = Logger.getLogger(classOf[SparkFetcher])
+  val fetchFailedTasks : Boolean = Option(fetcherConfigurationData.getParamMap.get(FETCH_FAILED_TASKS)).getOrElse("false").toBoolean
```

Comment:
`fetchFailedTasks` to `doFetchFailedTasks`

Reply:
Done.